lib/logstash/outputs/opsgenie.rb

```diff
@@ -240,4 +284,11 @@ def populateCommonContent(params, event)
     params['note'] = event.get(@noteAttribute) if event.get(@noteAttribute)
   end
 
+  private
+  def exponentialBackoff(retryCount)
+    nap = (2**retryCount - 1) / 2
```

Comment:
Assuming this is in seconds but might be worth renaming:

```suggestion
    seconds_to_sleep = (2**retryCount - 1) / 2
```

Reply:
Done